README.md

```diff
@@ -1317,6 +1317,9 @@ Other Style Guides
     numbers.forEach(num => sum += num);
     sum === 15;
 
+    const sum = numbers.map(num => num + 1);
```

Comment:
Sorry! I was taking the example from the issue itself. I realize this was the incorrect place to put it. I will close this PR.